src/electron/common/unified-feature-flags.ts

```diff
@@ -44,5 +45,14 @@ export function getAllFeatureFlagDetailsUnified(): FeatureFlagDetail[] {
             isPreviewFeature: false,
             forceDefault: false,
         },
+        {
+            id: UnifiedFeatureFlags.atfaResults,
+            defaultValue: false,
+            displayableName: 'Show ATFA results',
+            displayableDescription:
+                'Show the Accessibility Test Framework for Android results in Needs review',
```

Comment:
"in Needs review" seems to imply that the values could be displayed _somewhere else than Needs review_ if the flag were true. Removing "in Needs review" removes that ambiguity and is a complete description of what the flag does